package.json

```diff
@@ -107,5 +107,6 @@
       "**/test/**"
     ]
   },
-  "dependencies": {}
+  "dependencies": {},
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Does this matter?

Reply:
Not sure. It's added automatically by this version of yarn. Maybe we can turn that off in the package.json settings so it doesn't propose doing that.